manager/manager.go

```diff
@@ -1041,8 +1056,7 @@ func (m *manager) destroyContainer(containerName string) error {
 		return err
 	}
 
-	// Remove the container from our records (and all its aliases).
-	m.containers.Delete(namespacedName)
+	// Remove aliases from our records.
```

Comment:
It used to keep the record if error happened. So I think there will be a retry if `Stop` failed. I am wondering if we need to keep this logic in some form? I didn't confirm that retry will happen, but brief look seems to indicate it